genomedata/__init__.py

```diff
@@ -494,6 +453,131 @@ def vars(self):
             square(self.means)
 
 
+class _ChromosomeList(object):
+    def __init__(self):
+        self.open_chromosomes = {}  # NB: maintained by subclass
+
+    def __getitem__(self, name):
+        """Returns an existing opened chromosome if it exists"""
```

Comment:
More documentation is needed here—I'm guessing the idea is that subclasses will implement `__getitem__()` for cases where the parent `_ChromosomeList` returns `None`

Reply:
Added more explaination